recipes/libcurl/all/conanfile.py

```diff
@@ -305,7 +368,13 @@ def _configure_autotools(self):
                 self._autotools.defines.append('_AMD64_')
 
             configure_args = self._get_configure_command_args()
-            self._autotools.configure(vars=autotools_vars, args=configure_args)
+
+            if self.settings.os == "iOS" and self.settings.arch == "x86_64":
+                # please do not autodetect --build for the iOS simulator, thanks!
+                self._autotools.configure(vars=autotools_vars, args=configure_args, build=False)
```

Comment:
is there a bug in conan that it doesn't properly deduce build for iOS simulator?
I am asking cause we already had similar PR for another package in the past

Reply:
maybe a bug, I do not know,  
it will generate a wrong `--build` argument for configure run, the default (auto detection) from automake gets it right , so it is better to not let conan detect the `--build` argument for the simulator, since it will get it wrong